3.html

```diff
@@ -0,0 +1,72 @@
+
+<!DOCTYPE html>
+<html lang='ja'>
+<head>
+	<meta charset='utf-8'>
+	<title>課題 3-1</title>
+    <style>
+        table th, h3 {
+          font-size: 18px;
+        }
+      </style>
+</head>
+    <body>
+        <h2>ToDoリスト</h2>
+
+        <div>
+            <input name="status" type='radio' checked> すべて
+            <input name="status"type='radio'> 作業中
+            <input name="status"type='radio'> 完了
+        </div>
+        <br>
+
+        <table id ="taskTable"> 
+            <tr>
+                <th>ID</th>
+                <th>コメント</th>
+                <th>状態</th>
+                <th></th>
+            </tr>
+        </table>
+        
+        <h3>新規タスクの追加</h3>
+        <input type='text' id='addform'> <button id='add'>追加</button>
+
+        <script>
+            // ------------------------ ここから初期変数等の定義 ------------------------------------
+	        //初期タスクリスト
+            const myTasks = [];
+            let idNum = 0
+        	
+            // ------------------------ ここからイベントリスナー------------------------------------
+	        // 追加ボタンのイベントリスナー
+            const addButton = document.getElementById('add');
+            addButton.addEventListener('click', addTask);
+           
+        	// ------------------------ ここから関数定義 ------------------------------------
+            // 新規タスクの”追加”ボタンを押したときの関数
+            function addTask(){
```

Comment:
無名関数が使われることも多くなってきていますので、覚えてきおきましょう。
```suggestion
            const addTask = () => {
```

Reply:
承知しました。今回の例では、addTaskの変数に値を代入するものではないため、無名関数のような形にしなくても良いのでは？と思い、そのままにしてあります！